fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/FDBExceptionsTest.java

```diff
@@ -86,12 +90,32 @@ public void loggableTimeoutException() {
     }
 
     @Test
-    public void wrapCheckedException() {
+    void wrapCheckedException() {
         Exception cause = createCheckedException();
         final String methodName = "createCheckedException";
         testWrappedStackTrace(cause, methodName);
     }
 
+    @Test
+    void wrapInterruptedException() {
+        final InterruptedException err = createInterruptedException();
+        RuntimeException wrappedErr = FDBExceptions.wrapException(err);
```

Comment:
This line used to fail with an `IllegalArgumentException` before this change was made (the same `IllegalArgumentException` that is referenced in the issue)